webclient/projekt/filters.py

```diff
@@ -665,6 +679,18 @@ def filtr_dokumenty_ident(self, queryset, name, value):
             | Q(casti_dokumentu__dokument__ident_cely__icontains=value)
         ).distinct()
 
+    def filter_pristupnost(self, queryset, name, value):
+        if Heslar.objects.get(ident_cely="HES-000865") in value:
```

Comment:
@pesikj Tady nelze dát natvrdo ident_cely. Musí se to brát z konstant (hesla_dynamicka.py)